cbench_runner.lua

```diff
@@ -1,14 +1,15 @@
 --- init tarantool db wor tests
 
-
-
 if #arg < 1 then
     print('Please specify engine [memtx] or [vinyl]')
     os.exit(1)
 end
 
 local engine = arg[1]
 
+local wal_mode
+local count
```

Comment:
It is not obvious, how you got a non-zero exit code. Please, mention the strict mode and that it is enabled by default in a Debug build of tarantool.

I would not mix a certain change that fixes the strict mode with the style changes that are spotted by luacheck.

Reply:
Looks resolved now.